### PR TITLE
Update make primitives

### PR DESCRIPTION
**Pull request by bukosabino**
https://github.com/Featuretools/featuretools/issues/55
https://github.com/Featuretools/featuretools/issues/341

### CODE REVIEW

File: featuretools/primitives/base/aggregation_primitive_base.py [hunk 1]
```diff
@@ -109,14 +110,16 @@ def make_agg_primitive(function, input_types, return_type, name=None,
         base_of_exclude (list[PrimitiveBase]): Blacklist of
             primitives that cannot have this primitive in input_types.
 
-        description (str): Description of primitive.
```
Comment (kmax12) [inline featuretools/primitives/base/aggregation_primitive_base.py]: let's keep description in as a default value

Comment (bukosabino) [reply featuretools/primitives/base/aggregation_primitive_base.py]: > let's keep description in as a potential argument

Fixed. Now description will be fixed in this order:

1 - description sent through argument. 
2 - if 1 is None: description will be the function's docstring.
3 - if 1 and 2 are None: description will be a default description.

**Comment by kmax12** [comment]
Two things before we can merge

1. It is missing test cases for the new functionality
2. can you merge in the primitive-refactor branch? that will get merged into master before this gets merged in

**Comment by bukosabino** [comment]
The work continue in a new PR: https://github.com/Featuretools/featuretools/pull/425

**Pull request closed without merging**